atomic_reactor/plugins/exit_push_floating_tags.py

```diff
@@ -22,20 +22,15 @@ class PushFloatingTagsPlugin(ExitPlugin):
     key = PLUGIN_PUSH_FLOATING_TAGS_KEY
     is_allowed_to_fail = False
 
-    def __init__(self, tasker, workflow, registries=None):
+    def __init__(self, tasker, workflow):
         """
         constructor
 
         :param tasker: DockerTasker instance
         :param workflow: DockerBuildWorkflow instance
-        :param registries: dict, keys are docker registries, values are dicts containing
-                           per-registry parameters.
-                           Params:
-                            * "secret" optional string - path to the secret, which stores
-                              login and password for remote registry
         """
         super(PushFloatingTagsPlugin, self).__init__(tasker, workflow)
-        self.manifest_util = ManifestUtil(workflow, registries, self.log)
+        self.manifest_util = ManifestUtil(workflow, None, self.log)
```

Comment:
self.registries = get_registries(self.workflow, {})
self.manifest_util = ManifestUtil(workflow, self.registries, self.log)

Reply:
`ManifestUtil` [already does that](https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/manifest_util.py#L33) though.

Reply:
ohhh nvm, manifestutil is already getting those registries from get_registries